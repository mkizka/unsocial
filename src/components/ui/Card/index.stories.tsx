import type { Meta, StoryObj } from "@storybook/react";

import { Card } from ".";

const meta: Meta<typeof Card> = {
  component: Card,
};

export default meta;

type Story = StoryObj<typeof Card>;

export const Default: Story = {
  args: {
    className: "h-20 w-40",
  },
};
