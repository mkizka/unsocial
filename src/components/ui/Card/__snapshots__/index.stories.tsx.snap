// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ui/Card Default smoke-test 1`] = `
<div class="h-20 w-full rounded bg-primary-light shadow">
</div>
`;
